### ISSUE TITLE
[Bug] Xamarin.Forms 4.5 ignores iOS status bar colour change

### ISSUE THREAD

**Issue by IeuanWalker** [xamarin/Xamarin.Forms#9790]
Updating from 4.4 to 4.5 iOS status bar colour is now getting ignored.

The only change to the iOS status in this version is this PR - #8791 

In my project I set the Status bar background colour, but this is now getting ignored after updating to 4.5.

```csharp
// style ios navbar
UIColor barTintColor = UIColor.FromRGB(5, 60, 61);
UIColor tintColor = UIColor.White;
UINavigationBar.Appearance.BarTintColor = barTintColor;
UINavigationBar.Appearance.TintColor = tintColor;
UINavigationBar.Appearance.TitleTextAttributes = new UIStringAttributes() { ForegroundColor = UIColor.White };
UINavigationBar.Appearance.Translucent = true;
```

**Comment by IeuanWalker** [comment]
I need to use version 4.5 because I'm getting a NotImplementExeception when using 4.4 (no exception in android), but everything is fine with 4.5 except for the status bar colour.

I'd be happy with a workaround for now if anyone knows of one?

**Comment by ValonK** [comment]
We have the exact same issue, how can a update break something like this?

**Comment by brentvvosselen** [comment]
We have the same issue as well.

**Comment by hartez** [comment]
[_9790 Repro.zip](https://github.com/xamarin/Xamarin.Forms/files/4278573/_9790.Repro.zip)

**Comment by IeuanWalker** [comment]
Thanks @jsuarezruiz for the fix, just wondering if there is any time frame that this would be released in?

**Comment by IeuanWalker** [comment]
@jsuarezruiz @samhouts would it be available in pre-release or the nightly build for me to test?

**Comment by IeuanWalker** [comment]
@rmarinho is there way i can try the fix now?

**Comment by rmarinho** [comment]
you can try nighly 4.7.0.425

**Comment by IeuanWalker** [comment]
@rmarinho thanks, i just tried the nightly build, and I'm getting the same issue.

**Comment by nhdanh** [comment]
Same issue when upgrade to lastest XF

**Comment by tbaggett** [comment]
I still see the white nav bar on iOS after updating to the latest XF 4.5.0.396 package that was released two days ago. I deleted the bin and obj folders and did a full rebuild of the project after updating and before testing.

Should XF 4.5.0.396 contain the fix for this issue?

**Comment by ngalicoco** [comment]
Please when is this coming out. It is not fixed in 4.5.0.396 ?

**Comment by Burgler-dev** [comment]
I can confirm that 4.5.0.396 still has the issue. Will there be fix for 4.5 @rmarinho ?

**Comment by enterprisemk** [comment]
4.5.0.495 - issue still not fixed

**Comment by ngalicoco** [comment]
Same here 4.5.0.495 - still not working for me

**Comment by nhdanh** [comment]
Same here 4.5.0.495 - still not working for me

**Comment by tbaggett** [comment]
@rmarinho can this issue be reopened? It is not fixed. Thank you.

**Comment by alexsotir** [comment]
It's definitely not fixed. I had to roll back to 4.4.0.991864 so that I can still set the color.

**Comment by samhouts** [comment]
Please try the package in https://github.com/xamarin/Xamarin.Forms/issues/9765#issuecomment-607956957 and let us know if that resolves the issue for you. Thanks!

**Comment by ValonK** [comment]
This bug is still active in 4.5.0.539

**Comment by dmarjoram** [comment]
Not fixed for me in 4.5.0.617

**Comment by samhouts** [comment]
@ValonK @dmarjoram Please open a new issue with a reproduction project if this is still not working after you have updated to latest. Thanks!
